pydeps/mf27.py

```diff
@@ -6,12 +6,13 @@
 )
 import warnings
 with warnings.catch_warnings():
-    warnings.simplefilter('ignore', PendingDeprecationWarning)
+    warnings.simplefilter('ignore', DeprecationWarning)
```

Comment:
Could handle both warnings instead of just one, let me know WYT.

Reply:
Both would be good for the people who have to support older Pythons.